packages/password/lib/constants.js

```diff
@@ -1,11 +1,20 @@
 const MIN_LENGTH = 20
 const MAX_LENGTH = 30
-const DEFAULT_PASSWORD_RULES = `minlength: ${MIN_LENGTH}; maxlength: ${MAX_LENGTH};`
+const REQUIRED_CHARS = '-!?$&#%'
```

Comment:
Should `MIN_LENGTH`, `MAX_LENGTH` and `REQUIRED_CHARS` all have the `DEFAULT_` prefix?

Reply:
yep, done